onlyoffice/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
we have the license file in all other extension folders as well, but IMO it should only be needed on the top level of the repository.